tools/vms-generator/utils/utils.go

```diff
@@ -513,7 +513,7 @@ func GetVMIWindows() *v1.VirtualMachineInstance {
 					Spinlocks: &v1.FeatureSpinlocks{Retries: &spinlocks},
 				},
 			},
-			Machine: v1.Machine{Type: "q35"},
```

Comment:
Why do you remove it from the windows machine?

Reply:
I had to move the Type from `"q35"` to `""` (in this case the default for each Architecture: q35 in amd64 and pseries for ppc64le) because for the Power architecture we don't support the q35 machine type, so it would fail in the `make generate` step with the following error:
`Failed to validate vmi spec: failed to admit yaml for vmi-windows: FieldValueInvalid at spec.domain.machine.type: spec.domain.machine.type is not supported: q35 (allowed values: [pseries*])`

So moving it to the default value doesn't causes any changes to amd64 and makes it work for ppc64le